indexing-service/src/main/java/org/apache/druid/indexing/common/task/MergeTaskBase.java

```diff
@@ -114,6 +114,80 @@ public boolean apply(@Nullable DataSegment segment)
     this.segmentWriteOutMediumFactory = segmentWriteOutMediumFactory;
   }
 
+  private static String computeProcessingID(final String dataSource, final List<DataSegment> segments)
```

Comment:
Could you please keep the old method arrangement so that it's visible what have changed in these methods?